crates/flux-fixpoint/src/big_int.rs

```diff
@@ -27,6 +27,13 @@ impl BigInt {
     pub const ZERO: BigInt = BigInt { sign: Sign::NonNegative, val: 0 };
     pub const ONE: BigInt = BigInt { sign: Sign::NonNegative, val: 1 };
 
+    pub fn is_negative(&self) -> bool {
+        self.sign == Sign::Negative
+    }
+
+    pub fn val(&self) -> u128 {
+        self.val
+    }
```

Comment:
I prefer if we don't expose the representation of integers. I'd add a formatting method like so

```rust
    pub fn fmt_sexp(self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        match self.sign {
            Sign::NonNegative => write!(f, "{}", self.val),
            Sign::Negative => write!(f, "(- {})", self.val),
        }
    }
```